nucypher/blockchain/eth/interfaces.py

```diff
@@ -326,6 +326,34 @@ def get_contract_by_name(self, name: str, upgradeable=False, factory=Contract) -
 
         return unified_contract
 
+    def call_backend_sign(self, account: str, message: bytes) -> str:
+        """
+        Calls the appropriate signing function for the specified account on the
+        backend. If the backend is based on eth-tester, then it uses the
+        eth-tester signing interface to do so.
+        """
+        provider = self.providers[0]  # TODO: Handle multiple providers
+        if isinstance(provider, EthereumTesterProvider):
+            address = to_canonical_address(account)
+            sig_key = provider.ethereum_tester.backend._key_lookup[address]
+            signed_message = sig_key.sign_msg(message)
+            return signed_message
+        else:
+            return self.w3.eth.sign(account, data=message)  # Technically deprecated...
```

Comment:
I think, can include some marker comment ("TODO", "XXX" or whatever) so that we can find it once the issue is solved in geth/parity